### DISCUSSION TITLE
Unable to create a branch in github repository

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65007] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

I am new to Github and started using it frequently from the last month onwards as I began working in a new team.

Recently, I found out that I cannot create a branch in a repository. Reason revealed that the owner of the repo should assign me as collaborator. But the person has left the org and I need to continue using it. 
I am using Databricks and failed to push the code as I was not the owner of the repo. 
Need help to solve the problem. 

Thanks

**Comment by arafajs** [comment]
Unfortunately, you cannot do that.
Try this:

- **Contact the owner of the repository and ask them to assign you as a collaborator.** If they are still willing to collaborate, they can do this by going to the repository settings and adding you as a collaborator.
- **If the owner of the repository is no longer available, you can try to contact another administrator of the organization**. They may be able to assign you as a collaborator or give you access to the repository in some other way.
- **If you are unable to contact the owner of the repository or another administrator, you may need to create a new repository.** This is not ideal, but it is the only way to continue working on the code if you cannot get access to the old repository.

**Comment by proksh** [comment]
Given the situation you've described, here's a step-by-step plan to resolve your issue:

**Contact the Current Owner or Organization Admin:**

- If the person has left the organization, there must be someone who has admin rights over the repositories. This person can add you as a collaborator or give you the necessary permissions.
- If the repository is under an organization on GitHub, there may be other owners or members with administrative privileges who can help.

**Fork the Repository:**

- As a workaround, if you don't have access to the main repository, you can create a fork of that repository. A fork is a copy of a repository that allows you to freely experiment with changes without affecting the original project.
- Once you've made your changes in your fork, you can then submit a pull request to the original repository. An owner or collaborator on the original repository can review your changes and merge them if they are satisfactory.

**Using Databricks with Forked Repository:**

- If you are pushing code to the repo from Databricks, you can change the Git remote URL in Databricks to point to your forked repository. Make your necessary changes there and push.
- Later on, as mentioned, you can open a pull request to get the changes merged to the main repository if and when you get access.

**Create a New Repository:**

- If all else fails and you can't get access to the original repo, you can create a new GitHub repository and push your Databricks code there.
- Inform your team members of the new repository location so everyone can sync their work.

**Repository Transfer:**

- If, by any chance, the original owner is reachable or there's a way to contact them, they can also transfer the repository ownership to another user or to the organization. This way, you can have more control over the repo settings, including adding collaborators.

**Backup:**

- If you have a clone of the repository or access to its contents, make sure to keep a backup. This ensures that, even if there are issues with permissions or repository access, you have the data and code safe.


**Lastly, consider reaching out to GitHub support if there are any complications regarding ownership, especially if it's critical for your team's work.**
